### ISSUE TITLE
Report of fatal error on post archive page.

### ISSUE THREAD

**Issue by nerrad** [roughsmoothengine/organize-series#32]
This was reported in the page source:

> <b>Catchable fatal error</b>: Object of class WP_Error could not be converted to string in <b>/public_html/wp-content/plugins/organize-series/orgSeries-admin.php</b> on line <b>294</b><br />

**Comment by nerrad** [comment]
Line 294 for Organize Series 2.5.7 corresponds with:

```
$column_content .= sprintf(__('Part %1$s of %2$s%6$s in the series <br/><a href="%3$s" title="%4$s">%5$s</a>', 'organize-series'), $series_part, $count, $series_link, $series_name, $series_name, $drafts_included);
```

Something in that is returning a WP_Error object instead of series information.

**Comment by nerrad** [comment]
Customer reports this fixes their issue so I'll try and get this merged in at some point.

**Comment by nerrad** [comment]
this was merged and released with 2.5.8
